bgpd/bgp_routemap.c

```diff
@@ -1382,14 +1382,22 @@ static route_map_result_t route_match_interface(void *rule,
 {
 	struct interface *ifp;
 	struct bgp_path_info *path;
+	struct bgp_node *rn;
```

Comment:
How does this commit work with route leaking routes?  Have you tested this?

Reply:
the interface should be in the bgp_orig pointer. good point.
I did not test it.
I will try to make a test to check if it is correct ( I focus on vrf-lite backend).